src/rpc_api/mod.rs

```diff
@@ -706,6 +708,78 @@ pub mod eth_api {
         }
     }
 
+    #[derive(Debug, Clone, Default)]
+    pub struct EthSyncingResult {
+        pub done_sync: bool,
+        pub startingblock: i64,
+        pub currentblock: i64,
+        pub highestblock: i64,
```

Comment:
Is there a reason for not using `snake_case`?

Reply:
To match lotus and save serde rename lines. : )

Reply:
We should use `serde` renames for consistency; otherwise, in some places, we may even use CamelCase...

Reply:
Not a blocker, though.

Reply:
@LesnyRumcajs Fixed. Please re-approve